src/System.Collections/tests/Generic/SortedSet/SortedSet.Generic.Tests.cs

```diff
@@ -60,22 +60,37 @@ public void SortedSet_Generic_Constructor_IEnumerable_Null()
         public void SortedSet_Generic_Constructor_IEnumerable_IComparer(EnumerableType enumerableType, int setLength, int enumerableLength, int numberOfMatchingElements, int numberOfDuplicateElements)
         {
             IEnumerable<T> enumerable = CreateEnumerable(enumerableType, null, enumerableLength, 0, 0);
+#if netcoreapp
             SortedSet<T> set = new SortedSet<T>(enumerable, GetIComparer());
             Assert.True(set.SetEquals(enumerable));
+#else
+            // There is a bug in desktop that when the comparer is null it is overriding it to Default, but in one of the usages is not using the global comparer but the one passed in to the constructor.
+            // This is being done to not loose coverage and keep track of this bug.
```

Comment:
I can't find any bug in with netfx-port-consider that I can link to this specific issue. Will open one to track this bug and link it to the test.